dev/NavigationView/NavigationView.cpp

```diff
@@ -1398,6 +1434,88 @@ void NavigationView::UpdateAdaptiveLayout(double width, bool forceSetDisplayMode
     }
 }
 
+void NavigationView::UpdatePaneLayout()
+{
+    if (!IsTopNavigationView())
+    {
+        auto availableHeight = 0.0;
+        if (const auto &paneContentRow = m_itemsContainerRow.get())
+        {
+            // 20px is the padding between the two item lists
+            availableHeight = paneContentRow.ActualHeight() - 28;
+        }
+        if (const auto &paneFooter = m_leftNavFooterContentBorder.get())
+        {
+            availableHeight -= paneFooter.ActualHeight();
+        }
```

Comment:
I'd prefer this be a lambda

Reply:
Ah, I see you edit it again further down. You could leave this as is then.

---
In reply to: [522381640](https://github.com/microsoft/microsoft-ui-xaml/pull/3602#discussion_r522381640) [](ancestors = 522381640)

Reply:
I mean, we could make that two lambdas, one for the total available height and one for the actual new height of the max items. Would that be a good idea?

Reply:
Actually, I think two lambdas make it more clear what we are trying to achieve here, I'll switch to that.